meshkernel/py_structures.py

```diff
@@ -401,27 +400,33 @@ class MakeGridParameters:
         Default is `10.0`.
         block_size_y (float, optional): The grid block size in y dimension, used only for squared grids.
         Default is `10.0`.
+        upper_right_x (float, optional): The x coordinate of the upper right corner.
+        Default is `0.0`.
+        upper_right_y (float, optional): The y coordinate of the upper right corner.
+        Default is `0.0`.
     """
 
     def __init__(
         self,
-        num_columns=3,
-        num_rows=3,
+        num_columns=0,
+        num_rows=0,
```

Comment:
The initialization here and in MeshKernel is now incoherent with MeshKernelNET. I'll fix it in GRIDEDIT-563.